lib/src/core/show_functions.dart

```diff
@@ -139,27 +143,21 @@ class NFShowFunctions {
                         children: <Widget>[
                           if (additionalActions != null)
                             Flexible(
-                              child: ButtonBar(
-                                buttonPadding: EdgeInsets.zero,
-                                alignment: MainAxisAlignment.start,
+                              child: OverflowBar(
```

Comment:
`ButtonBar` is deprecated, so I replaced with and `OverflowBar`, but there is a regression here:

`ButtonBar` used to shrink its children to their minimum size. OverflowBar doesn't do that and allows it's children to expand. This is a problem for `NFIconButton`s which contains an `Align` that likes to expand. I opted to fix it in Sweyer using an `IntrinsicWidth`.